python/ccf/ledger.py

```diff
@@ -595,14 +606,30 @@ def __init__(self, filename: str):
         super().__init__(filename)
         self._filename = filename
         self._file_size = os.path.getsize(filename)
-        super()._read_header()
 
-    def commit_seqno(self):
-        try:
-            return int(self._filename.split("committed_")[1])
-        except IndexError:
-            # Snapshot is not yet committed
-            return None
+        entry_start_pos = super()._read_header()
+
+        # Snapshots embed evidence receipt since 2.x
+        if self.is_committed() and not self.is_snapshot_file_1_x():
```

Comment:
We can now verify the integrity of a standalone snapshot from Python directly. Note that the top-level endorsement isn't verified. Should we add an argument here (and `read_ledger.py`) to specify the service CA to verify the full chain of trust?

Reply:
That would be nice, yes!

Reply:
Cool. I'll do that in a follow-up PR.